### ISSUE TITLE
Controller.from_port doesn't handle Docker Compose network

### ISSUE THREAD

**Issue by grvn-ht** [torproject/stem#127]
Hey Stem Team!
Really nice work ongoing here,

I would like to propose a new feature for:
stem.control.Controller.from_port(address, port)

Right now address needs to be an IPV4, however, inside docker-compose, you can call a container by it's name.

Therefor we could imagine a class who overwrites the @staticmethod from_port:

class Docker_Controller(Controller):
  @staticmethod
  def from_port(address = 'tor', port = 9051):
    import stem.connection
    control_port = stem.socket.ControlPort(address, port)
    return Controller(control_port)

Here is an implementation's example: 
https://github.com/grvn-ht/torproxy_docker_compose

Hope it will be of help,

Regards

**Comment by atagar** [comment]
Hi G-ht, I'm glad you like the library!

[I already added the feature you propose](https://github.com/torproject/stem/commit/1087ebf6d) (IPv6 support in the from_port() method) but it's not in any releases. I moved on from Tor so Stem has become unmaintained.

As discussed on [another ticket](https://github.com/torproject/stem/issues/126) you'll need to contact 'gk at torproject dot org' if you'd care for this to change.

Cheers! -Damian
